docs/source/en/task_summary.mdx

```diff
@@ -120,7 +144,17 @@ There are two approaches to depth estimation:
 >>> from transformers import pipeline
 
 >>> depth_estimator = pipeline(task="depth-estimation")
->>> depth_estimator("path/to/image/file.jpg")
+>>> depth_estimator(
+...     "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/pipeline-cat-chonk.jpeg"
+... )
+{'predicted_depth': tensor([[[ 0.7767,  0.8142,  0.8237,  ...,  2.3059,  2.3640,  2.3277],
```

Comment:
This doesn't really bring anything to the table, so let's not show the output here.